src/components/confirm/confirm.stories.mdx

```diff
@@ -74,6 +74,171 @@ const MyComponent = ({ isOpen, handleClose, handleConfirm }) => (
   </Story>
 </Preview>
 
+
+### Single action
+
+<Preview>
+  <Story name="single action" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+
+## `destructive`
+
+<Preview>
+  <Story name="confirm - destructive" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            destructive
+            cancelButtonType="tertiary"
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+            onCancel={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+## `disabledConfirm`
+
+<Preview>
+  <Story name="confirm - disabledConfirm" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            destructive
+            disabledConfirm
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+            onCancel={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+## `disabledCancel`
+
+<Preview>
+  <Story name="confirm - disabledCancel" parameters={{ chromatic: { disable: true }}}>
+    {() => {
+      const [isOpen, setIsOpen] = useState(false);
+      return (
+        <>
+          <Button onClick={ () => setIsOpen(!isOpen) }>
+            Open Confirm
+          </Button>
+          <Confirm
+            title="Are you sure?"
+            subtitle="Subtitle"
+            disabledCancel
+            open={ isOpen }
+            onConfirm={ () => setIsOpen(false) }
+            onCancel={ () => setIsOpen(false) }
+          >
+            Content
+          </Confirm>
+        </>
+      )
+    }}
+  </Story>
+</Preview>
+
+##  `cancelButtonType`
```

Comment:
Looks like you need to add `ml={2}` to this the sibling button in this example. 
<img width="386" alt="Screenshot 2021-02-08 at 12 10 14" src="https://user-images.githubusercontent.com/56251247/107218023-f3f1f300-6a06-11eb-89b0-15b6b43cc04a.png">